addons/hw_drivers/controllers/driver.py

```diff
@@ -302,10 +302,14 @@ def send_alldevices(self):
     def usb_loop(self):
         usb_devices = {}
         devs = core.find(find_all=True)
+        cpt = 2
         for dev in devs:
-            path =  "usb_%04x:%04x_%03d_%03d_" % (dev.idVendor, dev.idProduct, dev.bus, dev.address)
+            dev.identifier =  "usb_%04x:%04x" % (dev.idVendor, dev.idProduct)
+            if dev.identifier in usb_devices:
+                dev.identifier += '_%s' % cpt
+                cpt += 1
```

Comment:
shouldn't this counter be defined out of the usb_loop method? as a global variable? isn't it possible to have two times usb_xxx_1 if we go two times in usb_loop?

Reply:
we can't have 2 instance of odoo.
And we call only one time the manager in driver.py.
So normaly... we can't go two times in usb_loop. 
Your opinion ?

Reply:
but it is called by `run` right? so every 0.3sec we go in this method and reset the counter to 2